src/pages/guides/getting-started/concepts.mdx

```diff
@@ -2,96 +2,119 @@ export const description = 'Initial concepts of the Nitric framework'
+Using the `nitric up` command in the Nitric CLI will initiate a deployment. This command will build your code into containers, then run those containers and connect them to the Nitric CLI's deployment API. That API gathers the requirements of your project (resources, security and other configuration) and communicates them to your chosen provider. The provider takes care of translating those requirements into cloud-specific infrastructure and performing the deployment.
 
-A question we often get is _"aren't the clouds and their services different?"_ Yes, they are somewhat different, but not in any significant way that impacts functionality.
+![diagram](/docs/images/concepts/how-it-works-build.svg)
 
-To compete with each other, cloud platforms have ended up providing a set of functionally identical base services. Across those services, the biggest differences tend to be their interfaces and how they're deployed, secured and scaled.
+The containers that are built during deployment include a copy of the runtime implementation for the provider. The runtime server is a lightweight adapter that translates requests from the Nitric SDK to the cloud-specific APIs in your provider. It also listens for events/requests from the cloud provider, such as HTTP requests or messages on a topic, and forwards them to your application code in a common format.
 
-For that reason, you can think of Nitric to cloud providers, like an ORM to SQL databases. There are differences, but in general, most SQL databases are similar enough and ORMs provide enough abstraction and value, that they're worth using. It's the same with Nitric and cloud services; some features won't be directly accessible but, for most applications, it won't matter.
+![diagram](/docs/images/concepts/how-it-works-runtime.svg)
 
-If you do need direct access to a cloud API, nothing is preventing you from importing a cloud SDK in a specific function or container that needs that low-level control, while still maintaining portability elsewhere.
+# Project structure
+
+Application's built with Nitric don't have a strict project structure. However, we do provide templates for each supported language, which can be a good place to start when learning how to build with Nitric. There are only two types of files required by Nitric projects:
+
+- Project file, e.g. `nitric.yaml`
+- Stack files, e.g. `nitric-develop.yaml` - these files define deployment targets and unique deployment configuration
+
+## Project files
+
+The existence of this file, typically `nitric.yaml`, indicates that the directory contains a project built with Nitric. The file defines the project's name and how to locate code entrypoint (handlers). This allows Nitric to find and build the code that serves your APIs, events and schedules.
+
+A typical `nitric.yaml` file looks like this:
+
+```yaml {{ tag: 'nitric.yaml' }}
+name: my-project
+handlers:
+  - functions/*.ts
+```
+
```

Comment:
After we merge this lets create a project file reference and link it here once its done.